### PR TITLE
Fix typehint in FunctionsAPI.create() & bump to version 0.68.2

### PR DESCRIPTION
**Pull request by HMEiding**

### CODE REVIEW

File: cognite/experimental/_api/functions.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def create(
             env_vars (Dict[str, str]):              Environment variables as key/value pairs. Keys can contain only letters, numbers or the underscore character. You can create at most 100 environment variables.
```
Comment (haakonvt) [inline cognite/experimental/_api/functions.py:83]: Github wont let me comment on line 70, i.e. this one:
```py
The Python runtime version is 3.8.3.
```
...but I think this should be updated to say "default runtime is xyz unless overridden by 'runtime' argument" or similar?

Comment (HMEiding) [reply cognite/experimental/_api/functions.py:83]: Maybe the best would just be to remove it?

Comment (haakonvt) [reply cognite/experimental/_api/functions.py:83]: I agree. Especially since the default is listed under `runtime`

Comment (haakonvt) [reply cognite/experimental/_api/functions.py:83]: Oh you actually removed it 😅

File: cognite/experimental/data_classes/functions.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ class Function(CogniteResource):
         env_vars (Dict[str, str]): User specified environment variables on the function ((key, value) pairs).
         cpu (Number): Number of CPU cores per function. Defaults to 0.25. Allowed values are in the range [0.1, 0.6].
         memory (Number): Memory per function measured in GB. Defaults to 1. Allowed values are in the range [0.1, 2.5].
-        runtime (str): Runtime of the function. Defaults to "py38". Allowed values are ["py37", "py38", "py39"].
+        runtime (str): Runtime of the function. Allowed values are ["py37", "py38", "py39"]. The runtime "py3x" resolves to the latest version of the Python 3.x.y series.
```
Comment (haakonvt) [inline cognite/experimental/data_classes/functions.py:29]:
```suggestion
        runtime (str): Runtime of the function. Allowed values are ["py37", "py38", "py39"]. The runtime "py3x" resolves to the latest version of the Python 3.x.y series. Will default to "py38" if not specified.
```

File: cognite/experimental/_api/functions.py [hunk 3]
```diff
@@ -83,7 +81,7 @@ def create(
...
             cpu (Number, optional):                 Number of CPU cores per function. Allowed values are in the range [0.1, 0.6], and None translates to the API default which is 0.25 in GCP. The argument is unavailable in Azure.
             memory (Number, optional):              Memory per function measured in GB. Allowed values are in the range [0.1, 2.5], and None translates to the API default which is 1 GB in GCP. The argument is unavailable in Azure.
-            runtime (str, optional):                The function runtime. Valid values are ["py37", "py38", "py39", `None`], and `None` translates to the API default.
+            runtime (str, optional):                The function runtime. Valid values are ["py37", "py38", "py39", `None`], and `None` translates to the API default. The runtime "py3x" resolves to the latest version of the Python 3.x.y series.
```
Comment (haakonvt) [inline cognite/experimental/_api/functions.py:84]:
```suggestion
            runtime (str, optional):                The function runtime. Valid values are ["py37", "py38", "py39", `None`], and `None` translates to the API default which currently is "py38". The runtime "py3x" resolves to the latest version of the Python 3.x.y series.
```

**Review by haakonvt — APPROVED** [review]
I leave it up to you regarding stating default or not 😄

**Pull request merged**